pmm/pmm2-upgrade-portal-tests.groovy

```diff
@@ -13,7 +13,8 @@ void runStagingServer(String DOCKER_VERSION, CLIENT_VERSION, CLIENT_INSTANCE, SE
         string(name: 'SERVER_IP', value: SERVER_IP),
         string(name: 'NOTIFY', value: 'false'),
         string(name: 'DAYS', value: '1'),
-        string(name: 'ADMIN_PASSWORD', value: ADMIN_PASSWORD)
+        string(name: 'ADMIN_PASSWORD', value: ADMIN_PASSWORD),
+        string(name: 'ENABLE_EXPERIMENTAL_REPO', value: 'no')
```

Comment:
where do you exactly use this?

Reply:
We are setting up expertimental or testing repo later on in the job, so stagging instance should not have enabled any of them.

Reply:
but why do use no for default experimental?

Reply:
Because if we want to enable testing as default, we will be already experimental enabled and pipeline will fail.